src/main/java/school/lemon/changerequest/java/matrix/Matrix.java

```diff
@@ -0,0 +1,104 @@
+package school.lemon.changerequest.java.matrix;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 12.12.2016.
+ */
+public class Matrix {
+
+    private final int Rows; // count Rows
```

Comment:
Please follow java code convention. All fields, methods, method arguments should be named in lowerCamelCase style.